src/pages/ScreenRouting/mediaScreens/MediaSeries/MediaSeries.tsx

```diff
@@ -182,6 +182,10 @@ const MediaSeries: ScreenComponent<PlaylistItem> = ({ data: seriesMedia }) => {
 
   if (!seriesMedia || !series) return <ErrorPage title={t('series_error')} />;
 
+  const getURL = (toEpisode: PlaylistItem) => {
+    return mediaURL({ media: seriesMedia, episodeId: toEpisode.mediaid });
```

Comment:
This one is going to be a little different, but we still want to replace the `onCardClick`method.
```suggestion
    return addQueryParam(location, 'e', toEpisode.mediaid);
```

`location` should come from a useLocation hook, which you can put up by the `useSearchParams` hook.